install.sh

```diff
@@ -8,13 +8,14 @@ sudo apt-get clean
 sudo apt-get update
 echo "======================== Cleaned apt-get ======================"
 
-echo "======================= Installing gcc 5 ======================"
+echo "======================= Installing gcc 6 ======================"
 sudo add-apt-repository ppa:ubuntu-toolchain-r/test
 sudo apt-get update
-sudo apt-get install gcc-5
-sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-5 20
+sudo apt-get install gcc-6 g++-6
+VER=6 ; PRIO=50
```

Comment:
I'm not sure why this is necessary. Any reason we can't just use the values below?